proposals/nnnn-actors.md

```diff
@@ -589,12 +589,14 @@ By default, the mutable stored properties (declared with `var`) of an actor clas
 All actor classes conform to a new protocol `Actor`:
 
 ```swift
-protocol Actor: AnyObject {
-  func enqueue(partialTask: PartialAsyncTask)
-}
+protocol Actor: AnyObject { /* unspecified */ }
```

Comment:
👍 Great, so we can separately talk about these once executors pitch is out there...